grpc/client.go

```diff
@@ -25,33 +25,26 @@ func ClientSetup(c *cmd.GRPCClientConfig, tls *tls.Config, metrics clientMetrics
 	if tls == nil {
 		return nil, errNilTLS
 	}
+	if len(c.ServerAddresses) != 1 {
```

Comment:
This is a somewhat confusing naming convention now, can we change it back to `ServerAddress`.

Reply:
Also no longer needs to be a slice.

Reply:
This comes from `cmd.GRPCClientConfig`, which is parsed out of JSON configs at startup. To switch this we need to create a synonym field, then switch the prod configs, then delete the old field. I think that's probably too disruptive for this PR, and probably isn't even worth the effort in the long term to just correct a plural versus singular issue.

Reply:
I don;t think it's that disruptive, we've done this once before (when we initially switched from having a single address to having multiple) and it seemed to go relatively smoothly.

I'm not as much concerned about the singular/plural issue, having a slice where we only ever use a single element of (and don't fail when there is more than one element) seems like confusing technical debt that has no actual benefit in the short term.

Reply:
But we do fail when there is more than one element; that's the line you're commenting on.